pulp_ansible/app/serializers.py

```diff
@@ -963,6 +963,38 @@ class Meta(CollectionImportListSerializer.Meta):
         fields = CollectionImportListSerializer.Meta.fields + ("error", "messages")
 
 
+class CollectionVersionCopyMoveSerializer(serializers.Serializer):
+    """
+    Copy or move collections from a source repository into one or mor destinations.
+    """
+
+    collection_versions = DetailRelatedField(
+        required=True,
+        view_name_pattern=r"content(-.*/.*)-detail",
```

Comment:
We could probably use `view_name` here too.

Reply:
@mdellweg what's the right value to use for viewname? I couldn't figure it out, so I finally gave up

Reply:
That is a good question. I guess something like "content-ansible-collection_version-detail".